website/developer-docs/setup/full-dev-environment.md

```diff
@@ -2,21 +2,21 @@
 title: Full development environment
 ---
 
-## Backend
-
-To create a local development setup for authentik, you need the following:
-
-### Requirements
+## Requirements
 
 -   Python 3.10
 -   poetry, which is used to manage dependencies, and can be installed with `pip install poetry`
 -   Go 1.18
 -   PostgreSQL (any recent version will do)
 -   Redis (any recent version will do)
+-   Node 14 (or later)
```

Comment:
Also probably not a big different, but while we're at it, authentik now uses node 16